frontend/src/pages/VendorDashboard.jsx

```
@@ -0,0 +1,371 @@
+    if (sortOption === 'name-desc') return b.name.localeCompare(a.name);
+    if (sortOption === 'price-asc') return parseFloat(a.price) - parseFloat(b.price);
+    if (sortOption === 'price-desc') return parseFloat(b.price) - parseFloat(a.price);
+    if (sortOption === 'rating-desc') return (b.rating || 0) - (a.rating || 0);
+    if (sortOption === 'reviews-desc') return (b.reviews?.length || 0) - (a.reviews?.length || 0);
+    // Default: newest first (assuming there's a createdAt field)
+    return new Date(b.createdAt || 0) - new Date(a.createdAt || 0);
+  });
+
+  // Helper function to render stars
+  const renderStars = (rating) => {
+    const ratingValue = parseFloat(rating) || 0;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)